submodules/wizard/views/step-accountContacts.html

```diff
@@ -0,0 +1,58 @@
+{{#*inline "inputPartial"}}
+	<div class="span4">
+		<label for="{{name}}">{{coalesce i18n.label name}}</label>
+		<input class="input-block-level{{#compare mask "===" "phoneNumber"}} phone-number{{/compare}}" type="{{#if type}}{{type}}{{else}}text{{/if}}" name="{{name}}" id="{{name}}"{{#if (coalesce i18n.placeholder i18n.label)}} placeholder="{{coalesce i18n.placeholder i18n.label}}"{{/if}}{{#if required}} required{{/if}}{{#if mask}} data-mask="{{mask}}"{{/if}} value="{{lookupPath @root.data name}}">
+	</div>
+{{/inline}}
+
+<div id="step_account_contacts" class="wizard-step container-fluid">
+	<div class="step-title">
+		{{i18n.accountsApp.wizard.steps.accountContacts.title}}
+	</div>
+	<div class="step-body">
+		<form id="form_account_contacts" name="form_account_contacts">
+			<div class="step-section">
+			{{#with i18n.accountsApp.wizard.steps.accountContacts as |accountContactsI18n|}}
+				{{#with @root.i18n.accountsApp.wizard.steps.general as |generalI18n|}}
+				<div class="section-title">
+					{{accountContactsI18n.salesRep.title}}
```

Comment:
I believe the spec calls for the sales rep section to not be exposed for now.

Reply:
I think it says not to save that info yet.

> At the time of writing, this field should not be saved as we are unsure when to display the parent section.

Should I remove the section for the time being? If so, do I remove the related logic too?

Reply:
Just hide it for now, this way you shouldn't have to touch the validation logic.

Reply:
Done :+1: I added an `if` block in the template to not render the sales representative section for the time being.